airflow/www_rbac/views.py

```diff
@@ -1090,7 +1091,10 @@ def graph(self, session=None):
         dag = dagbag.get_dag(dag_id)
         if dag_id not in dagbag.dags:
             flash('DAG "{0}" seems to be missing.'.format(dag_id), "error")
-            return redirect('/')
+            return redirect('/admin/')
```

Comment:
I assume that was copied over, `www_rbac` doesn't use the `/admin` prefix

Reply:
Good point, the downsides of haring to maintain two at the moment